lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -128,13 +129,13 @@ import Data.Either
 import Data.Generics.Internal.VL.Lens
     ( (^.) )
 import Data.List
-    ( nub, sortOn, unzip3 )
+    ( nub, sortOn, unzip4 )
```

Comment:
I like how it started with just unzip2 and gradually increased. We still have 3 left, after which we have to stop further development :)